### PR TITLE
Creating documentation for [Get|Set]-CsTeamsSipDevicesConfiguration commandlets

### PR DESCRIPTION
**Pull request by anmandav**
Creating documentation for [Get|Set]-CsTeamsSipDevicesConfiguration commandlets

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: teams/teams-ps/teams/Get-CsTeamsSipDevicesConfiguration.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+```powershell
+Get-CsTeamsSipDevicesConfiguration
+```
+
+## DESCRIPTION
+
+This cmdlet is used to retrieve the organization-wide Teams SIP devices configuration which contains settings that are applicable to SIP devices connected to teams using Teams Sip Gateway.
+
+You must be a Teams Service Administrator or a Global Administrator for your organization to run the cmdlet.
```
Comment (get-itips) [inline teams/teams-ps/teams/Get-CsTeamsSipDevicesConfiguration.md:27]: What about Teams Administrator role?

File: teams/teams-ps/teams/Set-CsTeamsSipDevicesConfiguration.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+## DESCRIPTION
+
+This cmdlet is used to manage the organization-wide Teams SIP devices configuration which contains settings that are applicable to SIP devices connected to teams using Teams Sip Gateway.
+
+You must be a Teams Service Administrator or a Global Administrator for your organization to run the cmdlet.
+
+## EXAMPLES
+
+### Example 1
+In this example, Bulk SignIn is set to Disabled across the organization.
```
Comment (get-itips) [inline teams/teams-ps/teams/Set-CsTeamsSipDevicesConfiguration.md:36]: Please move the description under the code block (example https://learn.microsoft.com/en-us/powershell/module/teams/new-teamchannel?view=teams-ps) and here it should say "Enabled"

Comment (anmandav) [reply teams/teams-ps/teams/Set-CsTeamsSipDevicesConfiguration.md]: Done, moved and corrected it to Enabled

File: teams/teams-ps/teams/Get-CsTeamsSipDevicesConfiguration.md (hunk 1 above)
Comment (anmandav) [reply teams/teams-ps/teams/Get-CsTeamsSipDevicesConfiguration.md]: Added all the roles which can execute the command

File: teams/teams-ps/teams/Set-CsTeamsSipDevicesConfiguration.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Set-CsTeamsSipDevicesConfiguration
+   [-BulkSignIn <String>]
+   [-WhatIf]
+   [-Confirm]
+   [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet is used to manage the organization-wide Teams SIP devices configuration which contains settings that are applicable to SIP devices connected to teams using Teams Sip Gateway.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsSipDevicesConfiguration.md:29]:
```suggestion
This cmdlet is used to manage the organization-wide Teams SIP devices configuration which contains settings that are applicable to SIP devices connected to Teams using Teams Sip Gateway.
```

File: teams/teams-ps/teams/Set-CsTeamsSipDevicesConfiguration.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+```powershell
+Set-CsTeamsSipDevicesConfiguration -BulkSignIn "Disabled"
+```
+In this example, Bulk SignIn is set to Disabled across the organization.
+
+## PARAMETERS
+
+### -BulkSignIn
+Indicates whether Bulk SingIn in to Teams SIP Devices is enabled or disabled for the Common Area Phone(CAP) accounts across the organization. Possible values are 'Enabled' and 'Disabled'.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsSipDevicesConfiguration.md:52]:
```suggestion
Indicates whether Bulk SingIn into Teams SIP devices is enabled or disabled for the common area phone (CAP) accounts across the organization. Possible values are **Enabled** and '**Disabled**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @anmandav, kindly commit the minor suggested changes. Many thanks.

**Comment by anmandav** [comment]
> Copy review - @anmandav, kindly commit the minor suggested changes. Many thanks.

Done. Committed the suggested modifications.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by anmandav** [comment]
@yogkumgit could you merge this Pull Request?

**Pull request merged**